packages/pages/src/dev/dev.ts

```diff
@@ -26,22 +27,30 @@ const handler = async ({
   noGenFeatures,
   port,
 }: DevArgs) => {
-  if (!fs.existsSync("config.yaml")) {
-    console.warn(
-      "It looks like you’re using an older setup of a Pages repo. Please run ‘npx pages upgrade’" +
-        " to upgrade to the latest format. This will setup a new configuration file config.yaml" +
-        " and install some new required dependencies."
+  if (!isUsingConfig()) {
+    logWarning(
+      "It looks like you're using an older setup of a Pages repo. Please run `npx pages upgrade`" +
+        " to upgrade to the latest format. This will setup a new configuration file, config.yaml," +
+        " and install some new required dependencies.\n" +
+        "If you'd like to upgrade without using the new config, run `npx pages upgrade --noMigration`."
```

Comment:
It might be better to avoid contractions with user facing messages.

Nbd either way. I had some on the original message.

Reply:
What do you mean?

Reply:
Just changing "you'd" to "you would" and "you're" to "you are" looks slightly more professional